library/sources/http-server/createRequestListener.ts

```diff
@@ -102,9 +102,13 @@ function createOnFinishRequestHandler(
       })
     ) {
       agent.onRouteExecute(context);
+      // Only count the request if the route is discovered
+      agent.getInspectionStatistics().onRequest();
+    } else if (context && context.attackDetected) {
+      // Also count the request if an attack was detected, also if the route is not discovered
+      agent.getInspectionStatistics().onRequest();
```

Comment:
I prefer a single call to `onRequest(...)`, wdyt?
- easier to reason about
- if we move things around, it will less likely break

(I know the chance is small that we would call this twice because of the `else if`)